hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: patch
+
+This release rebuilds ``find()`` on top of ``@given`` in order to have more code in common.
+It should have minimal user visible effec.
```

Comment:
:func:`~hypothesis.find` on top of :func:`@given <hypothesis.given>`

"effec" -> "effect"